models/tipo_bien.go

```diff
@@ -19,6 +19,8 @@ type TipoBien struct {
 	Activo            bool      `orm:"column(activo)"`
 	FechaCreacion     time.Time `orm:"column(fecha_creacion);type(timestamp with time zone)"`
 	FechaModificacion time.Time `orm:"column(fecha_modificacion);type(timestamp with time zone)"`
+	Tipo_bien_padre   int       `orm:"column(tipo_bien_padre);null"`
```

Comment:
- El tipo de dato de pronto no es `int` sino `*TipoBien`
- Para que lo anterior funcione, faltaría agregar la [propiedad orm](https://beego.me/docs/mvc/model/models.md#one-to-many) para el tipo de relación, posiblemente sea agregar `rel(fk_tipo_bien_tipo_bien_padre)`